src/initial_eval.cc

```diff
@@ -684,6 +684,12 @@ ER EvalVisitor::visit(const ImmediateAssertionStatement &stmt)
 	return ER::Success;
 }
 
+ER EvalVisitor::visit(const TimedStatement &stmt)
+{
+	ER result = stmt.stmt.visit(*this);
```

Comment:
We can't access `SynthesisSettings` here unless we make a provision for it. It's okay to ignore timed statements here unconditionally as the initial visitor will be reworked later anyway (it will be folded into the generic procedural visitor)

Reply:
got it

Reply:
sorry for the delay in communicating, i was away last week.

i took another look at this last night and it's going to be messy. the problem is that ProceduralVisitor needs access to settings. But Procedural visitor is constructed in multiple contexts that then would also need access to settings and so on. Adding further constructor arguments, which are in some cases a pretty long list, to all of these seems  clunky. i'd be adding 10-20 lines of boilerplate to forward a bool to a single line of code. so i stopped for the moment to see if you had any thoughts.

do you have a better way to do this? is this what you want to see?

Reply:
The visitor already holds a reference to `NetlistContext`, I think we should add a way to get the settings from that, i.e. a new `SynthesisSettings &settings;` field

Reply:
I was in the area so I've made the change in c53bd23

Reply:
i was able to rebase on that change and it helped. i was also able to implement the feature in the ER EvalVistor easily enough (even if it is temporary)

i did add a 2nd commit that adds a basic .gitignore if you like it, other wish we can drop it.

Reply:
No, great, `.gitignore` looks useful